packages/web/src/components/Alert/_theme.scss

```diff
@@ -0,0 +1,19 @@
+@use "sass:map";
+@use '@tokens' as tokens;
+
+$typography: tokens.$body-medium-text-regular;
+
+$success-color: tokens.$emotion-success-default;
+$success-background-color: tokens.$emotion-success-background;
+$danger-color: tokens.$emotion-danger-default;
+$danger-background-color: tokens.$emotion-danger-background;
+
+$border-radius: 8px;
+$space-inner: tokens.$space-700;
+$space-outer: tokens.$space-600;
+$space-icon: tokens.$space-600;
+
+$emphasis-font-weight: map.get(tokens.$body-small-text-bold, 'mobile', 'font-weight');
+
+$icon-success: url('data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMjAiIGhlaWdodD0iMjAiIHZpZXdCb3g9IjAgMCAyMCAyMCIgZmlsbD0ibm9uZSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KPHBhdGggZD0iTTEwIDBDNC40OCAwIDAgNC40OCAwIDEwQzAgMTUuNTIgNC40OCAyMCAxMCAyMEMxNS41MiAyMCAyMCAxNS41MiAyMCAxMEMyMCA0LjQ4IDE1LjUyIDAgMTAgMFpNMTEgMTVIOVY5SDExVjE1Wk0xMSA3SDlWNUgxMVY3WiIgZmlsbD0iI0I3RDE2RCIvPgo8L3N2Zz4K');
```

Comment:
I think this should be done better. Color of the SVG icon is present. I think that icon should take the color from styles.

Reply:
When the SVG is not base64-ed, we can use Sass variables in it (linking to Bootstrap this time! 🎉):

```scss
url("data:image/svg+xml,<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 20 20'><path fill='none' stroke='#{$form-check-input-checked-color}' stroke-linecap='round' stroke-linejoin='round' stroke-width='3' d='m6 10 3 3 6-6'/></svg>") !default;
```

https://github.com/twbs/bootstrap/blob/main/scss/_variables.scss#L881

Reply:
Fixed.